infra/feast-operator/config/manager/manager.yaml

```diff
@@ -70,6 +70,11 @@ spec:
           capabilities:
             drop:
             - "ALL"
+        env:
+        - name: RELATED_IMAGE_FEATURE_SERVER
+          value: feast:latest
+        - name: RELATED_IMAGE_GRPC_CURL
+          value: grpc:latest
```

Comment:
maybe we need this for odh too -> https://github.com/feast-dev/feast/blob/master/infra/feast-operator/config/overlays/odh/params.env
we can do next PR

Reply:
agreed, but i was also wanting to make this configurable via Makefile var for things like our e2e CI, etc